plugin/trace/setup.go

```diff
@@ -100,7 +115,7 @@ func normalizeEndpoint(epType, ep string) (string, string, error) {
 
 	if epType == "zipkin" {
 		if !strings.Contains(ep, "http") {
-			ep = "http://" + ep + "/api/v1/spans"
+			ep = "http://" + ep + "/api/v2/spans"
```

Comment:
Trace weren't being accepted by my local zipkin server with 
```
Expected a JSON_V1 encoded list, but received: JSON_V2
```
Due to the update of the vendor in #4109

Reply:
So this probably fixes #4160 bug?

Reply:
So this means we no longer support v1, since the #4109 ? That needs to be documented.

Reply:
> So this probably fixes #4160 bug?
This fix the 400 that is mentioned in the response

There is still the issue of the warning in my comment bellow ( i think i need to do a replace of `.` to `0.0.0.0` to have it work as expected

> So this means we no longer support v1, since the #4109 ? That needs to be documented.

Yes v1 is not supported since #4109